tldw_Server_API/tests/Embeddings/test_media_embeddings_submission_semantics.py

```
@@ -76,41 +76,38 @@ def create_job(self, **kwargs):
+    assert response.status == "partial"
+    assert response.job_ids == ["job-123"]
+    assert response.submitted == 1
+    assert response.failed_media_ids == [456]
+    assert response.failure_reasons == ["media_id=456: RuntimeError"]
 
 
 @pytest.mark.asyncio
-async def test_generate_embeddings_batch_fails_when_job_id_missing(monkeypatch):
+async def test_generate_embeddings_batch_raises_when_all_enqueues_fail_before_any_success(monkeypatch):
     class _MissingIdAdapter:
         def create_job(self, **kwargs):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `create_job`

Add return type annotation: `NoReturn`

(ANN202)